src/app/modules/multi-factor-auth/multi-factor-auth.service.spec.ts

```
@@ -0,0 +1,270 @@
+};
+
+const mockFirestore = {
+  collection: jest.fn().mockReturnValue(mockFirestoreCollection),
+};
+
+jest.mock('firebase-admin', () => ({
+  firestore: () => mockFirestore,
+}));
+
+// Export the mock objects so tests can control them
+export { mockFirestoreDoc, mockFirestoreCollection, mockFirestore };
```

Biome (2.1.2)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)